Resources/Textures/Imperial/PiratesNewHorizon/alhocolessence.rsi/meta.json

```
@@ -0,0 +1,31 @@
+{
+    "version": 1,
+    "size":
+    {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "By Ham Maggotson",
```

```suggestion
    "copyright": "Made by "hammaggotson" (discord) for Imperal Space server Space Station 14",
```
вообще как таковой политики меты на империале я не видел, но на других серверах выгляди это так